javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/SwitchExprTest.java

```diff
@@ -100,4 +101,146 @@ public void nestedSwitchRecordPatternShouldResolve() {
         NameExpr name = Navigator.findNameExpression(cu, "s").get();
         assertEquals("java.lang.String", name.resolve().getType().describe());
     }
+
```

Comment:
It seems to me that the only useful tests are those that try to resolve an unnamed variable or a method call for which the parameter is an unnamed variable (although I don't know if this is allowed by the specification).